src/file.cpp

```diff
@@ -34,6 +34,9 @@ using namespace rive;
 			#define RIVE_FMT_U64 "%llu"
 			#define RIVE_FMT_I64 "%lld"
 		#endif
+	#elif defined(_WIN32)
+		#define RIVE_FMT_U64 "%lld"
+		#define RIVE_FMT_I64 "%llu"
```

Comment:
The PRIu64/PRId64 should have worked, but I guess it was an issue with compiling under MingW.

I think this further emphasizes that we should use another approach.